### ISSUE TITLE
[Documentation] Basic Plugin Documentation

### ISSUE THREAD

**Issue by pseudonym117** [ace-lol/Ace#27]
As far as I can tell, there is no simple example plugin (e.g. hello world), nor a list of available APIs to hook.

If this could be provided, there are a few things I personally would like to add as plugins if I could do it quickly.

If an exhaustive list of APIs is too large an undertaking, a short wiki-like page on how to find them would be nice too. I just dont know where to start my reverse engineering,

**Comment by molenzwiebel** [comment]
Yep. I would like to write a basic guide to developing Ace somewhere this week. Soon™.
